doc/tests/block.rst

```diff
@@ -0,0 +1,13 @@
+``block``
+=========
+
+.. versionadded:: 1.23
+    The ``block`` test was added in Twig 1.23.
+
+``block`` checks if a block is defined in the current context of the template.
+
+.. code-block:: jinja
+
+    {% if 'title' is block %}
```

Comment:
Now that I'm reading it in a template, using a test looks wrong to me: `'title'` is a string, not a block.

I was thinking about using a function instead like `has_block('title')`:

```jinja
{% if has_block('title') %}
    <title>{{ block('title') }}<title>
{% endif %}
```

What do you think?